lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,1713 @@
+{
```

Comment:
<p>Tady bych navrhnul, aby funkce měla příznačnější jméno (<code>get_uk_crime_data</code>) a hlavně argumenty a <code>return</code>. Třeba</p><p> </p><p> </p><p><code>def get_uk_crime_data(months):</code></p><p><code>&nbsp;&nbsp;&nbsp;data_list = []</code></p><p><code>&nbsp;&nbsp;&nbsp;...</code></p><p><code>&nbsp;&nbsp;&nbsp;return data_list</code></p><p> </p><p>A co kdyby to vracelo rovnou <code>DataFrame</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

Reply:
jméno jsem změnl a vraci df

Reply:
<p>Ještě prosím ten <code>data_list</code> vytvořit uvnitř funkce ;)</p>